### PR TITLE
Closes #16 and #17

### LINKED ISSUE #16: /var/local/apps/umpr/app/views/home.html.php on line 23" while reading response header from upstream,
``` php
2015/04/08 11:38:06 [error] 9276#0: *1244779 FastCGI sent in stderr: "PHP message: PHP Notice:  Undefined index: meta-description in /var/local/apps/umpr/app/views/layout/html_head.html.php on line 4
PHP message: PHP Notice:  Undefined index: meta-keywords in /var/local/apps/umpr/app/views/layout/html_head.html.php on line 5
PHP message: PHP Parse error:  syntax error, unexpected '[' in /var/local/apps/umpr/app/views/home.html.php on line 23" while reading response header from upstream, client: 67.224.89.122, server: umpr.a4word.com, request: "GET / HTTP/1.1", upstream: "fastcgi://unix:/var/run/php5-fpm.sock:", host: "umpr.a4word.com"
```

### PR DESCRIPTION
**Pull request by Nava2**
- Remove SimpleTest -- warning removed
- Remove meta keywords -- apparently it's unused by search engines now
- Add meta-description

### CODE REVIEW

File: app/views/layout/html_head.html.php [hunk 1]
```diff
@@ -1,8 +1,7 @@
 <meta charset="utf-8">
 <meta http-equiv="X-UA-Compatible" content="IE=edge">
 <meta name="viewport" content="width=device-width, initial-scale=1">
-<meta name="description" content="<?php echo $GLOBALS["viewables"]["meta-description"] ?>">
-<meta name="keywords" content="<?php echo $GLOBALS["viewables"]["meta-keywords"] ?>">
```
Comment (aforward) [inline app/views/layout/html_head.html.php:5]: I wonder if the warning was more that the $GLOBALS wasn't set

``` php
$GLOBALS["viewables"]["meta-keywords"]
```

Comment (Nava2) [inline app/views/layout/html_head.html.php:5]: It was that `meta-keywords` wasn't, I added description [here](https://github.com/umple-ucosp/umpr.ui/pull/18/files#diff-8fda5307a067372337ec4b9adb16b55fR6)

File: public/index.php [hunk 2]
```diff
@@ -3,6 +3,7 @@
 $GLOBALS["viewables"]['view'] = 'home';
 
 $GLOBALS["viewables"]['title'] = "Umpr - Imported Umple Repository";
+setg('meta-description', 'Umpr Imported Umple Model Repository');
```
Comment (aforward) [inline public/index.php:6]: There she is

File: public/_config.php [hunk 3]
```diff
@@ -2,6 +2,8 @@
 session_start();
 require_once("_autoload.php");
 
+$GLOBALS["viewables"]["meta-description"] = "A repoistory of software models imported into Umple.";
```
Comment (Nava2) [inline public/_config.php]: `repoistory` -> `repository`

Comment (Nava2) [inline public/_config.php]: I'm unsure, but is `meta-description` usually application wide or page specific?

**Pull request merged**